routing/cross_mwm_graph.cpp

```diff
@@ -101,6 +104,20 @@ void CrossMwmGraph::FindBestTwins(NumMwmId sMwmId, bool isOutgoing, FeatureType
   }
 }
 
+bool CrossMwmGraph::AreAllNeighborsWithCrossMwmSection(NumMwmId numMwmId,
+                                                       std::vector<NumMwmId> & neighbors)
```

Comment:
Сейчас метод выглядит так:
```c++
  /// \brief Fills |neighbors| with number mwm id of all loaded neighbors of |numMwmId| and
  /// sets |allNeighborsHaveCrossMwmSection| to true if all loaded neighbors have cross mwm section
  /// and to false otherwise.
void CrossMwmGraph::GetAllLoadedNeighbors(NumMwmId numMwmId,
                                          std::vector<NumMwmId> & neighbors,
                                          bool & allNeighborsHaveCrossMwmSection) const
{
  m2::RectD const rect = m_countryRectFn(m_numMwmIds->GetFile(numMwmId).GetName());
  allNeighborsHaveCrossMwmSection = true;
  m_numMwmTree.ForEachInRect(rect, [&](NumMwmId id) {
    if (id == numMwmId)
      return;
    MwmStatus const status = GetMwmStatus(id);
    if (status == MwmStatus::NotLoaded)
      return;
    if (status == MwmStatus::NoCrossMwmSection)
      allNeighborsHaveCrossMwmSection = false;

    neighbors.push_back(id);
  });
}
```

Если его бить по честному на 3 метода, то получим:
1. получение соседних mwm;
2. вычленение из соседних загруженных;
3. проверка, что загружены только новые mwm с cross_mwm секцией;

2 и 3 требует обращения для каждой mwm к методу GetMwmStatus(id);, что может потребовать времени. Не хотелось бы вызывать его дважды для всех соседей. Предлагаю оставить, как есть сейчас.